dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContainerLoader.java

```diff
@@ -131,11 +132,15 @@ private InputStream buildVelocity(Container container, String uuid, PageMode mod
                 .append("\"" + container.getMaxContentlets() + "\"")
                 .append(" data-dot-accept-types=")
                 .append("\"");
-            for (ContainerStructure struct : csList) {
+            Iterator<ContainerStructure> it= csList.iterator();
+            while (it.hasNext()) {
+                ContainerStructure struct = it.next();
                 try {
                     ContentType t = typeAPI.find(struct.getStructureId());
-                    editWrapperDiv.append(t.variable())
-                        .append(",");
+                    editWrapperDiv.append(t.variable());
+                    if(it.hasNext()) {
+                        editWrapperDiv.append(",");
+                    }
                 } catch (DotDataException | DotSecurityException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)